resources/lang/en/follows.php

```diff
@@ -28,8 +28,8 @@
         'followers' => 'mapping subscribers',
         'page_title' => 'mapper watchlist',
         'title' => 'mapper',
-        'to_0' => 'stop notifying me when this user uploads new beatmap',
-        'to_1' => 'notify me when this user uploads new beatmap',
+        'to_0' => 'stop notifying me when this user uploads new beatmaps',
+        'to_1' => 'notify me when this user uploads new beatmaps',
```

Comment:
```suggestion
        'to_0' => 'stop notifying me when this user uploads a new beatmap',
        'to_1' => 'notify me when this user uploads a new beatmap',
```

I think this reads better.

Reply:
Yea either is fine in my opinion. Let's go with your suggestion.